store/src/java/com/zimbra/cs/account/ldap/LdapProvisioning.java

```diff
@@ -8200,7 +8200,7 @@ else if (numSigs == 0)
         } catch (LdapEntryAlreadyExistException nabe) {
             throw AccountServiceException.SIGNATURE_EXISTS(signatureName);
         } catch (LdapException e) {
-            throw e;
+            throw ServiceException.FAILURE("unable to create signature: " + signatureName, e);
```

Comment:
Signature creation seems more specific to account, so could also create a new one of type AccountServiceException like 
`AccountServiceException.SIGNATURE_CREATION_FAILURE(signatureName)`

Reply:
I set the same exception as line 8207. Do we need to update both?
```
        } catch (ServiceException e) {
            throw ServiceException.FAILURE("unable to create signature: " + signatureName, e);
```

Reply:
No need to update there as we are catching ServiceException and throwing  same with respective  message with cause.

Reply:
Thanks @zimsuchitgupta for your comment.
I added `AccountServiceException.SIGNATURE_CREATION_FAILURE` and updated LdapProvisioning to throw it.